src/helpers.ts

```diff
@@ -257,24 +257,25 @@ export function getDurationInUs(duration: number) {
   return Math.trunc(duration * 1e6);
 }
 
-export function megabytesToBytes(megabytes: number) {
-  return megabytes * 1024 * 1024;
+export function megabitsToBytes(megabytes: number) {
+  return megabytes * 1024 * 1024 / 8;
 }
 
-export function bytesToMegabytes(bytes: number) {
-  return bytes / 1024 / 1024;
+export function bytesToMegabits(bytes: number) {
+  return bytes / 1024 / 1024 * 8;
 }
 
 export const DEFAULT_NETWORK_CONDITIONS: NetworkConditions = {
-  downloadThroughput: megabytesToBytes(5), // megabytes/second
-  uploadThroughput: megabytesToBytes(3), // megabytes/second
+  downloadThroughput: megabitsToBytes(5), // Devtools CDP expects format to be in bytes/second 
```

Comment:
CDP does not report any errors for floating points, and the tests pass as expected when passing with floating points. I did this manually, but happy to write an explicit test.

Reply:
Don't think we need it, it was more of a question and what users would expect. We could round it off or leave it as it is, I will let you take the call.

Reply:
I think it's fine. As far as user input, I think users will have a need to express mgbps as floating points for more precision. This will sometimes lead to floating points for the corresponding bytes value after applying the calculation. As long as CDP does not report an error, I think it's fine.